releases/stellar/2025-01-GMP-v1.0.0.md

```diff
@@ -0,0 +1,132 @@
+# STELLAR GMP v1.0.0
+
+|                | **Owner**                            |
+| -------------- | ------------------------------------ |
+| **Created By** | @ahramy <ahram@interoplabs.io>       |
+| **Deployment** | @RiceAndMeet <steven@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [Crates](https://crates.io/users/interoplabs-ci)
+- [Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases)
+
+## Background
+
+Changes in the release:
+
+1. This is the v1.0.0 initial GMP release.
+
+## Deployment
+
+Create an `.env` config.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAIN=stellar
+```
+
+Deploy Stellar Gateway
```

Comment:
Before deployment can be started, an initial stellar chain config needs to be added in the config. Showcase that